pkg/openapi/controller/api_datasets_service.go

```
@@ -110,3 +141,41 @@ func (s *DatasetsApiService) UpdateDataset(ctx context.Context, user string, dat
 
 	return openapi.Response(http.StatusNotImplemented, nil), errors.New("UpdateDataset method not implemented")
 }
+
+func (s *DatasetsApiService) checkComputeForDatasetByRegion(datasetRealm string, datasetInfo *openapi.DatasetInfo) error {
+	eligibleComputes, err := s.dbService.GetComputeIdsByRegion(datasetRealm)
```

There is no check on `err`. Is this safe or can the error be ignored? If so, you may need to change it to something like the following:

```
eligibleComputes, _ := s.dbService.GetComputeIdsByRegion(datasetRealm)
```